*** ./smali/com/android/internal/view/ActionBarPolicy.smali	2014-05-12 03:14:18.633623997 +0800
--- /home/xinxin520/patchrom/miuiv5/normandy/temp/new_smali/framework.jar.out/./smali/com/android/internal/view/ActionBarPolicy.smali	2014-05-12 03:12:37.137627967 +0800
***************
*** 227,232 ****
--- 227,236 ----
      .locals 1
  
      .prologue
+     const/4 v0, 0x1
+     
+     return v0
+     
      iget-object v0, p0, Lcom/android/internal/view/ActionBarPolicy;->mContext:Landroid/content/Context;
  
      invoke-static {v0}, Landroid/view/ViewConfiguration;->get(Landroid/content/Context;)Landroid/view/ViewConfiguration;
